doc/development_process/issues.rst

```diff
@@ -9,3 +9,54 @@ a reference to the feature by mentioning the corresponding Issue or pull-request
 
 At any time it should be possible to establish the origin of a change and the
 reason behind it by following the references in the code.
+
+
+Feature Tracking
+*****************
+For feature tracking we use Github labels to classify new features and
+enhancements. The following is the description of each category:
+
+Enhancement
+  Changes to existing features that are not considered a bug and would not a
+  block a release. This is an incremental enhancement to a feature that already
+  exists in Zephyr.
+
+Feature request
+  A request for a feature that is not on the roadmap yet that has not been
+  vetted yet and needs further discussion and details.
+
+Feature
+  A high level roadmap feature with a detailed proposal with an owner. Features
+  require a detailed architectural design and must go through an RFC process.
+
+The following workflow should be used to process features and roadmap items.
+
+Requesting new features is done using the Github issues system using the label
+``feature-request``. This is the formal way for asking for a new feature in
+Zephyr and a way to signal to the project interest in the feature and a
+readiness and willingness to implement the feature and deliver it in one of the
```

Comment:
I'd agree.  We shouldn't stop someone requesting a feature because they need it for their project, but aren't capable of implementing it themselves.  How about:
```
This is the formal way for asking for a new feature in
Zephyr and indicating its importance to the project.  Often,
the requester will have a readiness and willingness to drive implementation of
the feature in an upcoming release, and should assign the request to themselves.
If not though, an owner will be assigned after evaluation by the TSC. 
A feature request can also have a companion RFC with more details on the feature and a
proposed design or implementation.
```